### PR TITLE
BaSP-TG-87: Add/edit timesheets

### PR DESCRIPTION
**Pull request by AlejandroSantini**

### CODE REVIEW

File: src/components/TimeSheet/index.js [hunk 1]
```diff
@@ -50,199 +58,166 @@ function TimeSheetForm() {
...
-    }
+  useEffect(() => {
+    setEmployeeOptions(
+      employees.map(({ _id, firstName, lastName }) => ({
+        value: _id,
+        label: `${firstName} ${lastName}`
+      }))
+    );
+    setTasksOptions(tasks.map(({ _id, description }) => ({ value: _id, label: description })));
   }, []);
```
Comment (franco-marini) [inline src/components/TimeSheet/index.js:69]: Nice catch-up! But there are some cases where you won't have any information regarding the employees or the tasks.

What do you think to use some of the thunks that were created before? 

Also, could you separate this into 2 different useEffect?

File: src/components/TimeSheet/index.js [hunk 2]
```diff
@@ -50,199 +58,166 @@ function TimeSheetForm() {
...
-        setModalMessage('There was an error');
-        setRedirect(false);
-      }
-      console.log(data);
-    } catch (error) {
-      console.error(error);
-    } finally {
-      setShowModal(true);
+  const routeChange = () => {
+    let path = `/time-sheets`;
```
Comment (franco-marini) [inline src/components/TimeSheet/index.js:84]: If you don't change/modify/update the variable, you must use always `const`

**Review by franco-marini — CHANGES_REQUESTED** [review]
Magnificient work! Just some little comments regarding the dropdowns

File: src/components/TimeSheet/index.js [hunk 3]
```diff
@@ -50,199 +60,175 @@ function TimeSheetForm() {
...
-              ? showModal
-              : () => {
-                  setShowModal(!showModal);
-                }
-          }
-          setModalMessage={Children}
-        >
-          <div>
-            <Button text={'Ok'} handler={redirect ? routeChange : () => setShowModal(!showModal)} />
+      <h3 className={styles.h3}>{requestType === 'PUT' ? 'Update Admin' : 'Add Admin'}</h3>
```
Comment (Facund0Fernandez) [inline src/components/TimeSheet/index.js:140]: This should not be update/add admin.

**Review by Facund0Fernandez — DISMISSED** [review]

**Review by FiolAxel — DISMISSED** [review]

**Review by Facund0Fernandez — DISMISSED** [review]

**Review by franco-marini — DISMISSED** [review]

**Review by Gearyandres94 — DISMISSED** [review]

**Review by Gearyandres94 — APPROVED** [review]

**Review by Facund0Fernandez — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Pull request merged**